Sources/Kitura/staticFileServer/FileServer.swift

```diff
@@ -101,14 +104,31 @@ extension StaticFileServer {
                 return
             }
 
-            tryToServeWithExtensions(filePath, response: response)
+            if tryToServeWithExtensions(filePath, response: response) {
+                return
+            }
+
+            let isDirectFileAccess = filePath.split(separator: "/").last?.contains(".") ?? false
```

Comment:
Could we add a comment referencing the logic from Express that we are replicating here? (for future reference)